src/js/components/Button/stories/IconColor.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Add } from 'grommet-icons';
+
+import { Box, Button, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from '../../../utils';
+
+const customButtonIconColor = deepMerge(grommet, {
+  button: {
+    color: 'accent-2',
+  },
+});
+
+const CustomIconLabelColor = () => (
+  <Grommet theme={customButtonIconColor}>
+    <Box align="center" pad="large">
+      <Box align="center" pad="large" gap="small">
+        <Button icon={<Add />} label="Add" onClick={() => {}} primary />
+        <Button icon={<Add />} label="Add" onClick={() => {}} />
+        <Button icon={<Add />} label="Add" gap="xlarge" onClick={() => {}} />
+        <Button label="500px gap" gap="500px" onClick={() => {}} />
+      </Box>
+    </Box>
+  </Grommet>
+);
+
+const IconLabelColor = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      <Box align="center" pad="large" gap="small">
+        <Button icon={<Add />} label="Add" onClick={() => {}} primary />
+        <Button icon={<Add />} label="Add" onClick={() => {}} />
+        <Button icon={<Add />} label="Add" gap="xlarge" onClick={() => {}} />
```

Comment:
I think lines 34 & 35 can be replaced with showing different cases of only an icon and only a label and see how it behaves. so we can compare the use cases.